win_etw_provider/src/provider.rs

```diff
@@ -269,21 +266,21 @@ mod win_support {
         }
 
         match is_enabled_code {
-            evntrace::EVENT_CONTROL_CODE_ENABLE_PROVIDER => {
+            EVENT_CONTROL_CODE_ENABLE_PROVIDER => {
```

Comment:
On these machines, could you make one change please.  Change the `use ... Etw::*` to this:

```
use windows::Win32::System::Diagnostics::Etw::{self, *};
```

and change these match arms to:

```
match ... {
    Etw::EVENT_CONTROL_CODE_ENABLE_PROVIDER => ...
    Etw::EVENT_CONTROL_CODE_DISABLE_PROVIDER => ...
}
```

The reason is that this avoids one common problem I've seen with Rust, where match patterns that are supposed to be matching against a constant instead accidentally become "match everything with this named binding" match.  If the windows-rs crate changes how it projects names (which has happened a few times), then this would still compile but would silently fail.

Using a qualified path name like `Etw::FOO` prevents that from happening.